Migrations/widgets.recipe.json

```diff
@@ -0,0 +1,1161 @@
+                  "Sanitized": false,
+                  "Tokenized": false,
+                  "Template": null
+                }
+              }
+            }
+          ]
+        },
+        {
+          "Name": "Card",
+          "Settings": {},
+          "ContentPartFieldDefinitionRecords": [
+            {
+              "FieldName": "MediaField",
+              "Name": "Image",
+              "Settings": {
+                "DisplayName": "Image",
+                "Position": "0"
+              }
+            },
+            {
+              "FieldName": "TextField",
+              "Name": "CTALabel",
+              "Settings": {
+                "DisplayName": "CTA Label",
+                "Position": "2"
+              }
+            },
+            {
+              "FieldName": "TextField",
+              "Name": "CTAUrl",
+              "Settings": {
+                "DisplayName": "CTA Url",
+                "Position": "3"
+              }
+            },
+            {
+              "FieldName": "TextField",
+              "Name": "ImageAlternateText",
+              "Settings": {
+                "DisplayName": "Image Alternate Text",
+                "Position": "1",
+                "Editor": null,
+                "DisplayMode": null,
+                "Hint": "Description of the appearance and function of the image.",
+                "Required": false,
+                "ContentIndexSettings": {
+                  "Included": false,
+                  "Stored": false,
+                  "Analyzed": false,
+                  "Sanitized": false,
+                  "Tokenized": false,
+                  "Template": null
+                }
+              }
+            }
+          ]
+        },
+        {
+          "Name": "Banner",
+          "Settings": {},
+          "ContentPartFieldDefinitionRecords": [
+            {
+              "FieldName": "ContentPickerField",
+              "Name": "Menu",
+              "Settings": {
+                "DisplayName": "Menu",
+                "Position": "3",
+                "Editor": null,
+                "DisplayMode": null,
+                "Hint": "Main menu used by users to navigate the main sections of the site.",
+                "Required": false,
+                "Multiple": false,
+                "DisplayedContentTypes": ["Menu"],
+                "ContentIndexSettings": {
+                  "Included": false,
+                  "Stored": false,
+                  "Analyzed": false,
+                  "Sanitized": false,
+                  "Tokenized": false,
+                  "Template": null
+                }
+              }
+            },
+            {
+              "FieldName": "MediaField",
+              "Name": "Logo",
+              "Settings": {
+                "DisplayName": "Logo",
+                "Position": "0"
+              }
+            },
+            {
+              "FieldName": "MediaField",
+              "Name": "LogoSmall",
```

Comment:
The responsive media field currently doesn't have a dynamic mode where it will determine the sizes based on the images that are selected. Think once that's in place, then we should definitely change this to use the "ResponsiveMediaField".